pallets/base/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
A brief module doc comment would help to understand why this is called the base pallet.
- Is this the pallet where any future shared methods are supposed go to?
- In case there are any remaining methods in `identity` that are shared, should we move those here as well?

Reply:
Added a module doc. It's basically a utility/framework level pallet with stuff that could have gone into `frame_system`. I'm not sure we have such extrinsics, but we could take a look and see if there's stuff to move. Can you think of any?